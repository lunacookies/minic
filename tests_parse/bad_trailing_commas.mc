func main {
	a := [,]
	b := [,,]
	c := [,a,,,]
	d := [1, 2, 3,,]
}
