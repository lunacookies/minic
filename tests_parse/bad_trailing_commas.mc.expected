func main {
	a := [<missing>]
	b := [
		<missing>,
		<missing>,
	]
	c := [
		<missing>,
		a,
		<missing>,
		<missing>,
	]
	d := [
		1,
		2,
		3,
		<missing>,
	]
}
tests_parse:19..20: error: missing array element
tests_parse:29..30: error: missing array element
tests_parse:30..31: error: missing array element
tests_parse:40..41: error: missing array element
tests_parse:43..44: error: missing array element
tests_parse:44..45: error: missing array element
tests_parse:62..63: error: missing array element
